CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+* Integrating APIs with security from [@ohltyler](https://github.com/ohltyler)
```

Comment:
the changelog should list what's relevant to the consumer. as a consumer i honestly don't care who built a feature, it just clutters the changelog. listing devs here is good for their vanity (maybe? i personally don't care if my name isn't listed, if i want to show proof that i did something i just point to the commit :)). if i want to know who built a feature i'll check the commit history.

what would however make sense (and what others are doing, e.g. [wildfly](https://github.com/wildfly/wildfly/releases)) is to link to the ticket (or, i guess in the absence of a ticket, to the PR) which allows interested parties to follow that link to get more details (incl. who built something).

Reply:
I think recognition is important as a part of our community building.  

I'll play around with incorporating PR links - thanks for that wildfly example, that does have a refined look to it.

Reply:
> I think recognition is important as a part of our community building.

i agree (and not just for OpenSearch) - but IMHO it still isn't relevant in the changelog. recognition is given in the commit history (either by being the author or being listed in the commit-msg as `Co-Authored-By`), in the issues & PRs, possibly in the forum and community meetings. but i think here it's cluttering and might even have the opposite effect if you forget somebody (what if multiple people contributed to a PR and you only list the one who was the `Author` in the commit? or the `Committer` (who might well have nothing to do with the PR except for having rebased it)?)